system/include/webgpu/webgpu.h

```diff
@@ -807,6 +899,13 @@ typedef struct WGPUVertexBufferLayoutDescriptor {
     WGPUVertexAttributeDescriptor const * attributes;
 } WGPUVertexBufferLayoutDescriptor;
 
+typedef struct WGPUBindGroupLayoutDescriptor {
+    WGPUChainedStruct const * nextInChain;
+    char const * label;
+    uint32_t entryCount;
+    WGPUBindGroupLayoutEntry const * entries;
+} WGPUBindGroupLayoutDescriptor;
```

Comment:
Any reason this struct moved? I guess the order changed in Dawn? nit: let's try to keep the order like [webgpu.h](https://github.com/webgpu-native/webgpu-headers/blob/main/webgpu.h)

Reply:
I assume because of the dependency on `WGPUBindGroupLayoutEntry`, which depends on types that have been added which changed the order.
Note, the generator doesn't actually have to put `WGPUBindGroupLayoutDescriptor` after `WGPUBindGroupLayoutEntry` because it's a by pointer dependency and not by value, but those seem to be the current rules of the generator, I guess

Reply:
Right. No worries, we can clean this up another time if things aren't the same across different copies of webgpu.h.